scripts/check-runtime-versions.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Required minimum versions.
+REQUIRED_GO_VERSION=$(grep 'version:' .promu.yml | awk '{print $2}')
```

Comment:
`awk '/^go / {print $2}' go.mod` is an alternative (if we want to keep the check for Go)

Reply:
Yeah used this now.